lib/Transforms/TransformationPassPipelines.cpp

```diff
@@ -22,6 +22,19 @@ using namespace mlir;
 
 namespace llzk {
 
+struct FullPolyLoweringOptions : public mlir::PassPipelineOptions<FullPolyLoweringOptions> {
```

Comment:
nit: you can remove the `mlir::` prefix (here and below) since you have the "using namespace"